### PR TITLE
3092 improve spellcheck

### PR DESCRIPTION
**Pull request by utkkkarshhh**
improved spellcheck and consistency for spelling by adding "NodeJS and JavaScript" to "accept.txt" and others in "reject.txt"

### CODE REVIEW

**Comment by utkkkarshhh** [comment]
How to add entry in Changelog verifier? Also i've already sent an email with Plone Contributors Agreement to agreements@plone.org.

**Comment by stevepiercy** [comment]
@utkkkarshhh for change log see https://6.docs.plone.org/volto/developer-guidelines/contributing.html#create-a-pull-request

A human will review your PCA and notify you when you are approved within a few business days. Please post a comment here when you are approved. Thank you!

**Comment by utkkkarshhh** [comment]
I've been added as a contributor to the Plone Organisation.

File: news/3092.documentation [hunk 1]
```diff
@@ -0,0 +1 @@
+Added JavaScript and NodeJS to "styles\Vocab\Plone\accept.txt" and "Javascript, js, javascript, nodejs and node" to "styles\Vocab\Plone\reject.txt" to improve spellcheck and consistency of spellings.
```
Comment (stevepiercy) [inline news/3092.documentation:1]: Brevity is better. Also give yourself credit!

```suggestion
Added `JavaScript` and `NodeJS` as accepted spellings, and deviations of them as rejected spellings. @utkkkarshhh
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
Thank you @utkkkarshhh. I will merge as soon as the jobs all pass.

**Comment by stevepiercy** [comment]
@utkkkarshhh if you would like to ensure that this change also gets imported into the main documentation, would you please submit a PR in https://github.com/plone/documentation?

**Pull request merged**